orchestrate-containers/Ansible Scripts/ceph_master_setup.yml

```diff
@@ -0,0 +1,79 @@
+---
+- hosts: master
+  remote_user: root
+
+  vars_prompt:
+    - name: "CephIP"
+      prompt: "Please enter the Ceph public network address(e.g 192.168.100.0)"
+      default: "192.168.100.0"
+      private: no
+
+    - name: "MasterHostname"
+      prompt: "Please enter the Master Hostname(e.g master)"
+      default: "master"
+      private: no
+
+  tasks:
+    - name: adding apt-key for ceph
+      apt_key:
+        url: https://download.ceph.com/keys/release.asc
+        state: present
+
+    - name: adding ceph repo list
+      apt_repository:
+        repo: deb https://download.ceph.com/debian-jewel/ stretch main
+        state: present
+
+    - name: install ceph-deploy
+      apt:
+        name:
+            - ceph-deploy
+        state: present
+
+    - name: Creating my-ceph directory
+      file: path=~/my-ceph state=directory
+
+    - name: Checking if Ceph cluster has already been deployed
+      stat: path=~/my-ceph/ceph.conf
```

Comment:
The task is redundant. In the following `shell` task, use the `creates` argument to check for the presence of the file. Do the same for other redundant "file check" tasks in other scripts.